Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -489,7 +490,6 @@ void CookedTracker::process(const std::vector<Cluster>& clusters, std::vector<Tr
   LOG(INFO) << "CookedTracker::process() entry " << entry++ << ", number of threads: " << mNumOfThreads
             << FairLogger::endl;
 
-  int nClFrame = 0;
   Int_t numOfClustersLeft = clusters.size(); // total number of clusters
```

Comment:
Do we need this? In any case, since you use ``rof`` vector both for the input (cluster selection) and output (records for the tracks), in case you want to exit before the loop over the ROFs, the ``rof`` vector ``NROFEntries`` must be set to 0.